packages/lib-classifier/src/hooks/useSubjectJSON.js

```diff
@@ -0,0 +1,73 @@
+import { useEffect, useRef, useState } from 'react'
+
+import { findLocationsByMediaType } from '@helpers'
+import JSONData from '@store/JSONData'
+
+function getSubjectUrl(subject) {
+  let jsonLocation = {}
+  // Find locations that have a JSON MIME type.
+  // TODO: do we need to support txt file fallback?
```

Comment:
Any resolution to this TODO comment? I'm wary of leaving todos in a PR in case there's no follow-up to it in the future.

Reply:
It’s a reference to this open issue in the Python client. 
https://github.com/zooniverse/panoptes-python-client/issues/210

This TODO has been in the code for about 4 years now.

Reply:
PH-TESS still uses text subjects so it’s still an open issue.

Reply:
I think it’s originally from this PR: https://github.com/zooniverse/front-end-monorepo/pull/2038